RNMixpanel/RNMixpanel.m

```diff
@@ -46,18 +46,19 @@ -(Mixpanel*) getInstance: (NSString *)name {
     }
 }
 
+// setAppSessionProperties iOS
 // setAppSessionProperties iOS
 RCT_EXPORT_METHOD(setAppSessionPropertiesIOS:(NSDictionary *)properties) {
     if ([properties objectForKey:@"minimumSessionDuration"]) {
         NSNumber *minimumSessionDuration = properties[@"minimumSessionDuration"];
         long long int intValue = [minimumSessionDuration longLongValue];
-        mixpanel.minimumSessionDuration = intValue;
+        [Mixpanel sharedInstance].maximumSessionDuration = intValue;
```

Comment:
Shouldn't this be `minimumSessionDuration`?

Reply:
Yep!

Reply:
Sending an update to affect this on this PR